IPython/core/magics/code.py

```diff
@@ -694,7 +704,7 @@ def edit(self, parameter_s='',last_call=['','']):
                     self.shell.safe_execfile(filename, self.shell.user_ns,
                                              self.shell.user_ns)
 
-        if is_temp:
+        if is_temp :
```

Comment:
Do you want to remove the space here and squash so that git blame doesn't see you touching this line?